l10n_ch_qr_bill_transaction_id/tests/test_transaction_id.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+import time
+
+from odoo.addons.account.tests.account_test_classes import AccountingTestCase
+
+CH_IBAN = 'CH15 3881 5158 3845 3843 7'
+QR_IBAN = 'CH21 3080 8001 2345 6782 7'
+
+
+class TestTransactionID(AccountingTestCase):
+    def setUp(self):
+        super(TestTransactionID, self).setUp()
+        self.customer = self.env['res.partner'].create(
+            {
+                "name": "Partner",
+                "street": "Route de Berne 41",
+                "street2": "",
+                "zip": "1000",
+                "city": "Lausanne",
+                "country_id": self.env.ref("base.ch").id,
+            }
+        )
+        self.env.user.company_id.partner_id.write(
+            {
+                "street": "Route de Berne 88",
+                "street2": "",
+                "zip": "2000",
+                "city": "Neuchâtel",
+                "country_id": self.env.ref('base.ch').id,
+            }
+        )
+        self.invoice1 = self.create_invoice('base.CHF')
+
+    def create_invoice(self, currency_to_use='base.CHF'):
```

Comment:
```suggestion
    def create_invoice(self, currency_xmlid='base.CHF'):
```
or require to pass the recordset directly?

Reply:
recordset wouldn't help getting a default value.